llama_index/agent/openai_agent.py

```diff
@@ -58,6 +68,7 @@ def call_function(
         role=MessageRole.FUNCTION,
         additional_kwargs={
             "name": function_call["name"],
+            "raw_output": output,
```

Comment:
minor concern about adding it here: we actually grab the kwargs here and pass it into the openai endpoint. 

I don't know how the endpoints handles extra unexpected input (maybe it ignores it safely), but probably not good to abuse it.

Reply:
I pop it off later on (it will raise an error yea)